### PR TITLE
Add content/pt-br/gitops.md

### PR DESCRIPTION
**Pull request by MrErlison**
Signed-off-by: Mr. Erlison <sisal.cordame_0a@icloud.com>

Add content/pt-br/gitops.md

### CODE REVIEW

**Review by brunoguidone — APPROVED** [review]
LGTM

**Comment by edsoncelio** [comment]
Thanks for this PR @MrErlison looks good to me but for now I'll put this on hold because of a recommendation from the maintainers to do not localize the terms that are with the status Feedback Appreciated, sorry about that :(

**Pull request closed without merging**
